src/config/axios.ts

```diff
@@ -20,6 +34,11 @@ axiosInstance.interceptors.request.use(
 
     config.headers['X-Correlation-Id'] = uuidv4()
 
+    // If the request has a payload performs the trim on all its strings
+    if (config.data) {
```

Comment:
Controllare se in questo caso non sia meglio usare `isEmpty` di lodash per un check più profondo (es. oggetto vuoto)